deploy/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM fedora:26
```

Comment:
Confirm with @tallclair what base image we should use.

Reply:
This is a TODO. Wont fix for initial commit.

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/4